commit fa1ded4a5a30ef00990a892df6163ddfb9317728
Author: Paul Eggert <eggert@cs.ucla.edu>
Date:   Thu Mar 23 11:45:11 2023 -0700

    Lebanon springs forward later this year
    
    This year Lebanon springs forward April 20/21 not March 25/26.
    (Thanks to Saadallah Itani.)
    * NEWS: Mention this.
    * asia (Lebanon): This year only, spring forward is April 21.

diff --git a/NEWS b/NEWS
index cd5cafe3..152cbc1c 100644
--- a/NEWS
+++ b/NEWS
@@ -4,6 +4,9 @@
 
   Changes to future timestamps
 
+    This year Lebanon springs forward April 20/21 not March 25/26.
+    (Thanks to Saadallah Itani.)
+
     Starting in 2023, Egypt will observe DST from April's last Friday
     through October's last Thursday.  (Thanks to Ahmad ElDardiry.)
     Assume the transition times are 00:00 and 24:00, respectively.

diff --git a/asia b/asia
index 25592db8..dd06a5fd 100644
--- a/asia
+++ b/asia
@@ -2691,6 +2691,12 @@ Zone	Asia/Pyongyang	8:23:00 -	LMT	1908 Apr  1
 
 
 # Lebanon
+#
+# From Saadallah Itani (2023-03-23):
+# Lebanon too announced today delay of Spring forward from March 25 to April 20.
+# From Paul Eggert (2023-03-23):
+# https://www.mtv.com.lb/en/News/Local/1352516/lebanon-postpones-daylight-saving-time-adoption
+#
 # Rule	NAME	FROM	TO	-	IN	ON	AT	SAVE	LETTER/S
 Rule	Lebanon	1920	only	-	Mar	28	0:00	1:00	S
 Rule	Lebanon	1920	only	-	Oct	25	0:00	0	-
@@ -2713,9 +2719,11 @@ Rule	Lebanon	1988	only	-	Jun	1	0:00	1:00	S
 Rule	Lebanon	1989	only	-	May	10	0:00	1:00	S
 Rule	Lebanon	1990	1992	-	May	1	0:00	1:00	S
 Rule	Lebanon	1992	only	-	Oct	4	0:00	0	-
-Rule	Lebanon	1993	max	-	Mar	lastSun	0:00	1:00	S
+Rule	Lebanon	1993	2022	-	Mar	lastSun	0:00	1:00	S
 Rule	Lebanon	1993	1998	-	Sep	lastSun	0:00	0	-
 Rule	Lebanon	1999	max	-	Oct	lastSun	0:00	0	-
+Rule	Lebanon	2023	only	-	Apr	21	0:00	1:00	S
+Rule	Lebanon	2024	max	-	Mar	lastSun	0:00	1:00	S
 # Zone	NAME		STDOFF	RULES	FORMAT	[UNTIL]
 Zone	Asia/Beirut	2:22:00 -	LMT	1880
 			2:00	Lebanon	EE%sT
